daemon/logger/context.go

```diff
@@ -61,6 +61,23 @@ func (ctx *Context) ExtraAttributes(keyMod func(string) string) map[string]strin
 	return extra
 }
 
+// ParseLabels parses the labels context
+func (ctx *Context) ParseLabels(valueMod func(string) string) {
+	// No point in running this method if valueMod is nil
+	if valueMod == nil {
+		return
+	}
+
+	labels, ok := ctx.Config["labels"]
+	if ok && len(labels) > 0 {
+		for _, l := range strings.Split(labels, ",") {
+			if v, ok := ctx.ContainerLabels[l]; ok && strings.Contains(v, "{{.") {
```

Comment:
I thought it would be a waste to execute the template regardless to the value.
I will change that, though.